<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberWindow</class>
 <widget class="QDialog" name="MemberWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#MemberWindow {
	background: #333947
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>261</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="addToCartLabel">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#addToCartLabel {
	font-size: 16px;
	text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>Purchase Item:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cartItemCB">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox#cartItemCB {
	background: black;
	color: white;
	font-size: 12px;
	border:1px solid #FFFFFF;
	border-radius: 1px;
}

QComboBox#cartItemCB:hover {
	background: #202121;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="quantitySB">
      <property name="styleSheet">
       <string notr="true">QSpinBox#quantitySB {
	background: black;
	color: white;
	font-size: 12px;
}

QSpinBox#quantitySB:hover {
	background: #202121;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addToCartButton">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#addToCartButton {
	background: #121d2b;
	color: white;
	font-size: 16px;
	border:2px solid #FFFFFF;
	border-radius: 5px;
}

QPushButton#addToCartButton:hover {
	background: #374a63;
}</string>
      </property>
      <property name="text">
       <string>Add to Cart</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="cartTW">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>total</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="purchaseButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#purchaseButton {
	background: #121d2b;
	color: white;
	font-size: 16px;
	border:2px solid #FFFFFF;
	border-radius: 5px;
}

QPushButton#purchaseButton:hover {
	background: #374a63;
}</string>
   </property>
   <property name="text">
    <string>Purchase</string>
   </property>
  </widget>
  <widget class="QLabel" name="cartTotalLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>255</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#addToCartLabel {
	font-size: 16px;
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>Hello, </string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#logout_button{
	background: gray;
	color: black;
	font-size: 16px;
	border:2px solid #34362d;
	border-radius: 5px;
}

QPushButton#logout_button:hover{
	background: #b2b9bf;
}</string>
   </property>
   <property name="text">
    <string>Logout Now</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
